### DISCUSSION TITLE
Dimmension units in gerber output

### DISCUSSION THREAD

**Discussion by tox104** [INTI-CMNB/KiBot#286] [Q&A] [answered]
How to get dimensions in gerbers to be in mm?
Gerber units are mm.

**Comment by set-soft** [comment, marked as answer]
Hi @tox104 !
KiCad 6 has some "magic dimensions" that are computed by KiCad (not fixed).
These dimensions can be set to a particular unit, but can be automatic (I think this is the default).
The Python API can't control it, but KiBot can adjust them using the global option `units`. So try to define `units: millimeters` in the global section.

**Reply by tox104** [reply]
Thank you, for some reason yesterday global options did not work for me, but I cant provide any details.
This morning updated Ubuntu on WSL added global section with `units: millimeters` and now output is correct.
